Brewfile

```diff
@@ -1,9 +1,18 @@
-brew 'python@2'
+brew 'pyenv'
```

Comment:
Since `brew` is platform-specific, I'd rather not include this. I think most of this config should live in `pyproject.toml` anyway which then platform-specific implementations can interpret?

Reply:
What platform specific implementation are you referring to? I don't really understand which tooling is supposed to read from pyproject.toml

Reply:
> I'd rather not include this.

What, the Brewfile? Or `brew 'pyenv'`.

Reply:
>What platform specific implementation are you referring to? I don't really understand which tooling is supposed to read from pyproject.toml

I'm just saying we should not invent yet another place for this information. I don't know if such tooling exists but at least we have a defined spec. We can write our own tool to read this information and use it instead of reading it from a random file.

Reply:
>What, the Brewfile? Or brew 'pyenv'.

The `Brewfile`.

Reply:
The names of those homebrew packages are in and of itself platform specific. How are you going to abstract over those?

Reply:
Well then, I guess I cannot. I was hoping these C libraries had standardized names across platforms.

Reply:
Telling people to run `brew bundle` and keeping all the system packages required for sentry, in sentry's Brewfile, is the best thing that can be done right now.